tests/bluetooth/tester/src/btp_csip.c

```diff
@@ -61,6 +108,11 @@ static void csip_discover_cb(struct bt_conn *conn,
 
 	LOG_DBG("Found %zu sets on member[%u]", set_count, conn_index);
 
+	memcpy(sirk, member->insts[0].info.set_sirk, sizeof(member->insts[0].info.set_sirk));
```

Comment:
In line 123 you do `cur_csis_inst = &member->insts[0];`; suggest to do this before this line, so you can do
```suggestion
	memcpy(sirk, cur_csis_inst->.info.set_sirk, sizeof(cur_csis_inst->info.set_sirk));
```

Reply:
Done